package.json

```diff
@@ -59,16 +59,16 @@
     ]
   },
   "dependencies": {
-    "@arcgis/core": "^4.26.5",
+    "@arcgis/core": "^4.27.6",
     "@dnd-kit/sortable": "^7.0.2",
     "@fortawesome/fontawesome-free": "^6.4.0",
     "@fortawesome/fontawesome-svg-core": "^6.4.0",
     "@fortawesome/free-solid-svg-icons": "^6.4.0",
     "@fortawesome/react-fontawesome": "^0.2.0",
-    "bootstrap": "^5.2.3",
+    "bootstrap": "^5.3.0",
```

Comment:
**question**: does this require the dark mode import?

Reply:
Yes. Their definition of a breaking change does not align with mine: https://github.com/twbs/bootstrap/issues/38683

Reply:
don't you need to add it to the sass then?

Reply:
Sorry, I got my projects mixed up. This one does not need the explicit export because it [imports all of bootstrap](https://github.com/agrc/wfrc-wasatch-choice/blob/main/src/main.jsx#L2).